CommonTools/RecoAlgos/interface/RecoTrackSelectorBase.h

```diff
@@ -19,6 +19,8 @@ class RecoTrackSelectorBase {
     ptMin_(cfg.getParameter<double>("ptMin")),
     minRapidity_(cfg.getParameter<double>("minRapidity")),
     maxRapidity_(cfg.getParameter<double>("maxRapidity")),
+    meanPhi_((cfg.getParameter<double>("minPhi")+cfg.getParameter<double>("maxPhi"))/2.),
+    rangePhi_((cfg.getParameter<double>("maxPhi")-cfg.getParameter<double>("minPhi"))/2.),
```

Comment:
Ah, ok. Would the checks I mentioned above (`minPhi < maxPhi`, `minPhi < π`, `maxPhi > -π`) be enough (i.e. the first case would give an error), or the input to be normalized/sanitized first?

To me a configuration would be easier to understand (by me) if `minPhi < maxPhi` would be enforced despite of phi wrapping around, but if the opposite is desired, I'm fine with that as well.